Find a functionally equivalent code:android.print.PrintManager.getPrintJob:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Gets a print job given its id.
 *
 * @return The print job list.
 * @see PrintJob
 * @hide
 */

@@ -1,4 +1,8 @@
 {
+    if (mService == null) {
+        Log.w(LOG_TAG, "Feature android.software.print not available");
+        return null;
+    }
     try {
         PrintJobInfo printJob = mService.getPrintJobInfo(printJobId, mAppId, mUserId);
         if (printJob != null) {

